components/phenotypic-feature-table.js

```diff
@@ -0,0 +1,69 @@
+// node_modules
+import Link from "next/link";
+import PropTypes from "prop-types";
+// components
+import { DataGridContainer } from "./data-grid";
+import SortableGrid from "./sortable-grid";
+
+const phenotypicFeaturesColumns = [
+  {
+    id: "phenotypic_feature_id",
+    title: "Phenotypic Feature",
+    display: ({ source }) => {
+      return (
+        <Link href={source["@id"]}>
+          {source.notes ? source.notes : source["@id"]}
```

Comment:
this is incorrect it needs to use phenotypic_features.feature.term_name plus the values in phenotypic_features